src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx

```diff
@@ -66,17 +66,14 @@ const LocationPreferences = () => {
             dataTestId="openToRelocationCity"
           />
         </div>
-        <InputFormError error={touched.remoteOnly && errors.remoteOnly}>
-          <SwitchInput
-            onBlur={handleBlur}
-            id="remoteOnly"
-            checked={values.remoteOnly}
-            label="I’m looking for remote jobs only"
-            onChange={handleChange}
-            name="remoteOnly"
-            dataTestId="remoteOnly"
-          />
-        </InputFormError>
+        <SwitchInput
+          id="remoteOnly"
+          checked={values.remoteOnly}
+          label="I’m looking for remote jobs only"
+          onChange={handleChange}
+          name="remoteOnly"
```

Comment:
**nitpick:** we should have enum with inputs names

Reply:
I'll fix it 👍